d3-array/d3-array-tests.ts

```diff
@@ -330,12 +330,23 @@ mergedArray = d3Array.merge(testArrays); // inferred type
 mergedArray = d3Array.merge<MixedObject>(testArrays); // explicit type
 // mergedArray = d3Array.merge<MixedObject>([[10, 40, 30], [15, 30]]); // fails, type mismatch
 
+// cross() ---------------------------------------------------------------------
+
+let crossed: Array<[string, number]>;
+
+crossed = d3Array.cross<string, number>(['x', 'y'], [1, 2]);
```

Comment:
Sounds reasonable. Consider adding a separate test if you feel that it'd be useful. Otherwise the change looks good to me.